util/localpool/localpool_test.go

```diff
@@ -21,24 +21,17 @@ import (
 	"sync"
 	"testing"
 
-	. "github.com/pingcap/check"
+	"github.com/stretchr/testify/require"
 )
 
 type Obj struct {
 	// nolint:unused
 	val int64 // nolint:structcheck // Dummy field to make it non-empty.
 }
 
-func TestT(t *testing.T) {
-	TestingT(t)
-}
-
-var _ = Suite(&testPoolSuite{})
-
-type testPoolSuite struct {
-}
+func TestPool(t *testing.T) {
+	t.Parallel()
 
-func (s *testPoolSuite) TestPool(c *C) {
 	numWorkers := runtime.GOMAXPROCS(0)
```

Comment:
Is there any other paralleled test units will modify the GOMAXPROCS and affect this test？

Reply:
I think not in this binary. But it is a good point we run it in serial. Maybe postpone to when we actually suffer.

Reply:
I think it will be more robust with serial excution. Could you delete it？

Reply:
updaed